clothes_scrap/__init__.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import time
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+service = Service('C:\\Users\\Richie\\Downloads\\chromedriver_win32\\chromedriver.exe') 
+options = Options()
+options.add_argument('--headless')  
+driver = webdriver.Chrome(service=service, options=options)
+
+driver.get('https://scrapingclub.com/exercise/list_infinite_scroll/')
+wait = WebDriverWait(driver, 10)
+wait.until(EC.presence_of_element_located((By.CLASS_NAME, 'col-lg-4.col-md-6.mb-4')))
```

Comment:
Very good use of expected condition 💯! 👍 

This becomes very necessary when scraping pages with long and uncertain wait times and works when you know for sure which dom element state is there in "good condition"